<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchoClient</class>
 <widget class="QWidget" name="EchoClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="lyServer">
     <item>
      <widget class="QLabel" name="lbServerName">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leServerName">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="lyText">
     <item>
      <widget class="QLabel" name="lbText">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leText">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbPort">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sldPort">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdPort"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pbSend">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="ln">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QListView" name="lvInfo"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
